src/pages/auth/reset-password/confirm.page.tsx

```diff
@@ -23,16 +25,18 @@ const SignupConfirmPage = () => {
     if (email == null) router.push(`${baseAuthPath}/reset-password`);
   });
 
-  const { mutateAsync: requestResetPassword } = usePostAuthReset({
-    onSuccess(_, variables) {
+  const [, { isSuccess }] = useRequestPassword();
+
+  useValueChanged(isSuccess, () => {
+    if (isSuccess) {
       timer.reset();
       timer.start();
     }
   });
 
   const handleResend = async () =>
-    requestResetPassword({
-      body: { email_address: email, callback_url: window.location.origin + `${baseAuthPath}/reset-password/` }
+    requestPasswordReset({
```

Comment:
This method from the connection should simply accept the email address and callback URL directly. The component code doesn't need to know how they're going to be used, that's the job of the Connection.

In general, in v3:
 * Components declare what their requirements are by using connections. They are just trying to say "I need a method to trigger a password reset, and I need to know if it worked"
 * Connections are responsible for providing the requirements that a component might need, and part of that is providing methods that interface with the API layer. 
 
Components should ideally not know or care about two things: where the data they require came from, and what the particulars are of updating those data.